includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -1894,7 +1894,10 @@ public static function print_url_as_title( $post ) {
 	 *
 	 * @return string Title.
 	 */
-	public static function filter_the_title_in_post_list_table( $title, $post ) {
+	public static function filter_the_title_in_post_list_table( $title, $post = '' ) {
+		if($post == ''){
```

Comment:
@KZeni I found that this function is a callback for `the_title` hook. In that can we need to update the following thing:
1. The second parameter for `the_title` filter in `int` type, so set default value to `0`: https://core.trac.wordpress.org/browser/tags/5.0/src/wp-includes/post-template.php#L158
2. According to point 1, you can also update `$post` param value type to `int` instead of `WP_Post` in function description comment block.
3. Use trinary condition ` $post = empty( $post ) ? get_post() : $post`

Reply:
I found we can simply further. Since `$post` is only used by `get_post_type()`, and since this function takes `null|int|WP_Post` as its type, then we just have to provide an empty default value for `$post`.